components/AddHabitsForm.js

```diff
@@ -55,6 +55,7 @@ function AddHabitForm() {
     longestStreak: 0,
     currentStreak: 0,
     lastUpdated: null,
+    score: 0,
```

Comment:
Usually, you wouldn't need to already specify all the keys in your initial state object. You can initialize this as an empty object, if you want to :)